### ISSUE TITLE
Challenges - ONGOING

### ISSUE THREAD

**Issue by D3Portillo** [Arena-Mayhem/frontend#18]
## TL;DR

Allow users to see a list of active challenges

## Component Story

- Adapt current challenge item to look similar to the Figma version
- As a user in a mobile device I want the challenge item to adapt without breaking the UI

## Dev Notes

- Check for date ago, if not existent you can use https://date-fns.org format or formatRelative
- The date input is a timestamp (seconds I think) so we must format to ms for JS to recognize this Date - (`secs * 1000`)

## References

![Image](https://github.com/user-attachments/assets/5607ffcc-e199-41eb-972d-00880bfd39df)

### Mobile

![Image](https://github.com/user-attachments/assets/94b9c50b-1668-46f6-ad2d-805c1b06a5d0)

## Resources

- Figma View: https://www.figma.com/design/csb1dd9yjNAg4sWTmVEZww/Arena-Mayhem?node-id=770-17744&t=dWfCkQG4RMaA9HeQ-0

---

> [!TIP]
> To relative format a date you can use `date-fns` formatDistance function https://date-fns.org/v4.1.0/docs/formatDistance
